woff1/tools/validator/validator.py

```diff
@@ -593,7 +594,7 @@ def _testHeaderFlavor(data, reporter):
     - If the directory cannot be unpacked, the flavor can not be validated. Issue a warning.
     """
     header = unpackHeader(data)
-    flavor = header["flavor"]
+    flavor = header["flavor"].decode()
```

Comment:
Binary data needs to be decoded in order to be able to perform string comparison in Python 3.